x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result.scss

```diff
@@ -0,0 +1,65 @@
+.appSearchResultContainer {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  width: 100%;
+}
+
+.appSearchResult {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+
+  &__contentWrap {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    padding: $euiSize;
+    display: grid;
+    grid-template-rows: 1fr $euiSize * 1.25 /* 20px */;
+    border-right: 1px solid $euiColorLightShade;
+  }
+
+  &__contentInner {
+    display: grid;
+    grid-gap: 7px;
+    overflow: hidden;
+  }
+
+  &__fieldsetContainer {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__hiddenFieldsIndicator {
+    font-size: $euiSizeM;
+    color: $euiColorDarkShade;
+    align-self: end;
+    padding-top: 0;
+  }
+
+  &__toggleExpandButton {
+    justify-self: stretch;
+    align-self: stretch;
+    display: flex;
+    width: $euiSize * 3.125; // 50px
+
+    &:hover,
+    &:focus,
+    &:focus,
+    &:active {
+      outline: none;
```

Comment:
[Not a change request, just a general note/me thinking out loud] I'm generally a little leery of `outline: none` - it's usually inadvisable from an accessibility POV for keyboard users, etc. That being said we are replacing the focus state w/ a background color change, so this scenario's probably fine

Reply:
OK, I'll test this out, maybe we'll just drop the focus none.

Reply:
88eddf3512ffd3305f1a9a815f22a4613e9a0042